deep_net.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+
+import pickle
+import random
+from time import time
+import numpy as np
+import tensorflow as tf
+import tensorflow.keras.layers as layers
+from sklearn.metrics import accuracy_score
+from snippets import get_random_parameters
+
+def train_neural_net(train_set, test_set, label, training_time, store_intermediate_results=False):
```

Comment:
Using python 3 (as all person of good taste should)? Consider using [type hints](https://docs.python.org/3/library/typing.html).